docs/conf.py

```diff
@@ -268,4 +268,7 @@ def setup(app):
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
```

Comment:
Probably worth removing this comment line, as this isn’t an example configuration any more.